lib/facter/fastx_version.rb

```diff
@@ -1,5 +1,7 @@
 Facter.add('fastx_version') do
   setcode do
+    cmd = "/bin/false" # Default to nothing
```

Comment:
Will this cause issues if Puppet gets non-zero? Doesn't `/bin/false` return 1, by default?

Reply:
Good point, let's use `/bin/true` instead